### PR TITLE
Add crs_ttl_inst_protect table, new in LOL-3.22b

### PR DESCRIPTION
**Pull request by strk**
This is NOT using patches but rather a
CREATE IF NOT EXISTS. I find it better to deal with,
for comments and such...

### CODE REVIEW

**Comment by strk** [comment]
Switching to use of patches for this we're going to miss many advantages, like the fact that we're using a global GRANT for ALL TABLES IN SCHEMA, which would NOT be catching the table added via patch. What advantaged would we have with using patches ?

**Comment by strk** [comment]
What's your opinion on this matter @palmerj ?
Now that we have `linz-bde-schema-load` script, which is meant to both install and upgrade, why should we insist in using dbpatch for anything that can be done without ? Remember we even have a version string to tell whether or not we upgraded...

**Comment by strk** [comment]
ok I just noticed I'm missing a test for table being versioned, I suspect using patched is better to deal with that, will do patches

**Comment by strk** [comment]
It looks like our upgrade tester does NOT fail with current code, so we must be NOT testing upgrades made without the --revision switch. The `bde_processor` script does indeed NOT pass `--revision` when upgrading the database, but only does right after loading the very first dataset. This would be a problem at the moment, unless we change `linz-bde-schema-load` to enable versioning on all tables IFF table versioning is enabled on any table. This would be indeed similar to enabling version on a new table (Created via dbpatch) when other tables are versioned.

**Comment by strk** [comment]
I went with patch. Can you re-review now @imincik ?

File: sql/99-patches.sql [hunk 1]
```diff
@@ -241,5 +241,58 @@ $$
...
+      TO bde_user;
+
+  -- If tables are versioned, version the new table
+  IF EXISTS (SELECT p.oid FROM pg_catalog.pg_proc p,
+                               pg_catalog.pg_namespace n
+                          WHERE p.proname = 'ver_is_table_versioned'
+                          AND n.oid = p.pronamespace
+                          AND n.nspname = 'table_version')
+  THEN
+    IF table_version.ver_is_table_versioned('bde', 'crs_work')
```
Comment (imincik) [inline sql/99-patches.sql:285]: * @strk , why do you test if `crs_work` is versioned ? Just testing some random table if it is versioned ? Is it going to work if tables are installed without versions first and then after first L0 upload versions are added ?

Comment (strk) [reply sql/99-patches.sql:285]: Yes, this is just a random table. I'm following what was already done in linz-lds-bde-schema here: https://github.com/linz/linz-lds-bde-schema/blob/1.10.1/sql/90-patches.sql#L111

If versioning is enabled later, it should include versioning of the new table (versioning is enabled for all tables under the `bde` schema: https://github.com/linz/linz-bde-schema/blob/1.10.1/sql/versioning/01-version_tables.sql#L46)

**Pull request merged**